client/scss/utilities/_root-scope-classes.scss

```diff
@@ -92,6 +100,14 @@
   text-decoration: none;
 }
 
+.plain-list {
+  @include plain-list();
```

Comment:
Worth refactoring this out similar to the font things so we just have this on the elements that currently include it?